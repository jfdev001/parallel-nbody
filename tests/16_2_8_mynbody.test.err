Reservation number 3910865: Reserved 16 hosts for 900 seconds 
Run on 16 hosts for 960 seconds from Mon Jan 16 17:18:19 2023
: node013/0 node013/1 node014/0 node014/1 node015/0 node015/1 node016/0 node016/1 node017/0 node017/1 node018/0 node018/1 node057/0 node057/1 node058/0 node058/1 node059/0 node059/1 node060/0 node060/1 node061/0 node061/1 node062/0 node062/1 node063/0 node063/1 node064/0 node064/1 node065/0 node065/1 node066/0 node066/1 
Running N-body with 8 bodies and 1000 steps

N-body took: 0.159 seconds
Performance N-body: 0.00 GFLOPS
