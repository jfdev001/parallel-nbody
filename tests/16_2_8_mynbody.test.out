   313.023      9.014      1.302      1.397      4.576      2.838
   875.979    766.024    -32.474    -25.195     -4.301      4.844
   512.017    214.989   -207.071   1013.766      3.398     -2.226
   234.008     45.986    203.539     90.508      1.619     -2.844
   842.002    746.979     68.020  -1324.000      0.428     -4.247
   502.013    248.995   7322.375   -747.432      2.649     -1.026
   921.994    187.981 -90917.239 144519.826     -1.170     -3.831
   865.008    280.009  83561.549 -143528.871      1.689      1.866
STATUS=PASSED
